gio/src/io_extension.rs

```diff
@@ -0,0 +1,57 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use glib::translate::*;
+use glib::Type;
+use std::fmt;
+use std::ptr;
+
+/// The implementation of an `IOExtensionPoint`.
+#[doc(alias = "GIOExtension")]
+#[derive(Copy, Clone, Eq, PartialEq)]
+pub struct IOExtension(ptr::NonNull<ffi::GIOExtension>);
+
+impl fmt::Debug for IOExtension {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "IOExtension {}", self.name())
```

Comment:
```suggestion
        f.debug_struct("IOExtension")
           .field("name", &self.name())
           .field("priority", &self.priority())
           .finish()
```
Would do something like this

Reply:
Cool, I learned something new :)